docs/antora.yml

```
@@ -1,12 +1,12 @@
 name: docs
 title: Payara Platform
-version: 5.24.0
+version: master
```

This had been set to 5.24.0 (as seen on the left) but is usually 'master' and left unchanged in release notes PRs - I've changed it back to master but not sure if that's correct and it should be a version increment instead - if it is wrong, please make a suggested change review with correct value

---

Thats right, I think it was changed by accident

---

I don't really know my antora, but I assume this would be that it displays the _current_ master? Rather than the release notes at the point of release (the tag)

---

This is a value that is used by Antora in the version selector and in URLs. The reason why it used to be set to "master" is to be able to generate the docs for version branches and the master branch together and switch between them to compare them. We don't need to do that anymore because we use Netlify to preview all our PRs and branches. It's more practical if we change the version in this release notes PR so that we don't need to change it in the version branch later.

We already set the version field in the release notes PRs in the Enterprise docs and I didn't see that anybody would complain about that. So let's keep it consistent and let's update the version field in every release notes PR:

```suggestion
version: 5.25.0
```